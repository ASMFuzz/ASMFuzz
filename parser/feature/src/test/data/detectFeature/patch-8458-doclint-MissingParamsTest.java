@@ -1,6 +1,6 @@
 /*
  * @test /nodynamiccopyright/
- * @bug 8004832
+ * @bug 8004832 8285496
  * @summary Add new doclint package
  * @modules jdk.javadoc/jdk.javadoc.internal.doclint
  * @build DocLintTester
@@ -21,4 +21,7 @@ void missingParam(int param) { }
 
     /** . */
     <T> void missingTyparam() { }
+
+    /** . */
+    public class MissingTyparam<T> { /** . */ MissingTyparam() { } }
 }