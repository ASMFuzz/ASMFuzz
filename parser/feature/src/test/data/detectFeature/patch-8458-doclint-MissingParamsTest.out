@@ -10,5 +10,8 @@ MissingParamsTest.java:20: warning: no @param for param
 MissingParamsTest.java:23: warning: no @param for <T>
     <T> void missingTyparam() { }
              ^
-4 warnings
+MissingParamsTest.java:26: warning: no @param for <T>
+    public class MissingTyparam<T> { /** . */ MissingTyparam() { } }
+           ^
+5 warnings
 